src/test/common/fdc3.app-channels.ts

```diff
@@ -173,7 +177,7 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
         const testChannel = await cc.createTestChannel()
         const resolveExecutionCompleteListener = cc.initCompleteListener(acTestId10)
 
-        cc.openChannelApp(acTestId10, undefined, APP_CHANNEL_AND_BROADCAST_TWICE, 2)
+        await cc.openChannelApp(acTestId10, testChannel.id, APP_CHANNEL_AND_BROADCAST_TWICE, 2)
         await resolveExecutionCompleteListener;
 
         const context = await testChannel.getCurrentContext("fdc3.instrument");
```

Comment:
These test should be extended to also check that adding a context listener does not auto-replay the current context.